src/panel/poi/blocks/Stars.jsx

```diff
@@ -0,0 +1,35 @@
+/* global _n */
+import React from 'react';
+import Block from './Block';
+import { IconStar } from 'src/components/ui/icons';
+import { ACTION_BLUE_BASE } from 'src/libs/colors';
+import { capitalizeFirst } from 'src/libs/string';
+import { useI18n } from '../../../hooks';
```

Comment:
You can use the alias `src/hooks` like you did with the lines before, that way it's easier to move files.